spec/rspec/matchers/built_in/change_spec.rb

```diff
@@ -223,7 +223,7 @@ def ==(other)
   it "warns if passed a block using do/end instead of {}" do
     expect do
       expect {}.to change do; end
-    end.to raise_error(SyntaxError, /block passed to the `change` matcher/)
+    end.to raise_error(ArgumentError, /Block not received by the `change` matcher/)
```

Comment:
To me a `SyntaxError` is really profound; it means we haven't even been able to convert a text file to a ruby program yet. Certainly there is a concept of proper Rspec syntax, but `SyntaxError` is Ruby's thing. I'd be totally comfortable with something like `RspecSyntaxError`.